contracts/kernel/Kernel.sol

```diff
@@ -47,52 +41,44 @@ contract Kernel is IKernel, KernelStorage, Initializable {
     /**
     * @dev Creates a permission that wasn't previously set. Access is limited by the ACL.
     *      if a created permission is removed it is possible to reset it with createPermission.
-    * @notice Create a new permission granting `_entity` the ability to perform actions of role `_role` on `_app` (setting `_parent` as parent)
+    * @notice Create a new permission granting `_entity` the ability to perform actions of role `_role` on `_app` (setting `_owner` as parent)
     * @param _entity Address of the whitelisted entity that will be able to perform the role
     * @param _app Address of the app in which the role will be allowed (requires app to depend on kernel for ACL)
     * @param _role Identifier for the group of actions in app given access to perform
-    * @param _parent Address of the entity that will be able to revoke the permission. If set to `_entity`, then it will be able to grant it too
+    * @param _owner Address of the entity that will be able to grant and revoke the permission further.
     */
     function createPermission(
         address _entity,
         address _app,
         bytes32 _role,
-        address _parent
-    ) auth(CREATE_PERMISSIONS_ROLE) external
+        address _owner
+    )   auth(CREATE_PERMISSIONS_ROLE)
+        external
```

Comment:
This slightly deviates from the [official style guide](https://solidity.readthedocs.io/en/develop/style-guide.html#function-declaration); do you like aligning the modifiers like this?

The official way would be:

```
function createPermission(
    address _entity,
    address _app,
    bytes32 _role,
    address _parent
)
    auth(CREATE_PERMISSIONS_ROLE)
    external
{ ... }
```

Some other places ([L75](https://github.com/aragon/aragon-core/pull/156/files#diff-14a0803379fe8527d47915e3727fe7d0R75), [L95](https://github.com/aragon/aragon-core/pull/156/files#diff-14a0803379fe8527d47915e3727fe7d0R95), and [L113](https://github.com/aragon/aragon-core/pull/156/files#diff-14a0803379fe8527d47915e3727fe7d0R113)) are similar.